packages/postcss-normalize-url/src/index.js

```diff
@@ -1,12 +1,29 @@
 import path from 'path';
 import valueParser from 'postcss-value-parser';
 import normalize from 'normalize-url';
-import isAbsolute from 'is-absolute-url';
 
 const multiline = /\\[\r\n]/;
 // eslint-disable-next-line no-useless-escape
 const escapeChars = /([\s\(\)"'])/g;
 
+// Scheme: https://tools.ietf.org/html/rfc3986#section-3.1
+// Absolute URL: https://tools.ietf.org/html/rfc3986#section-4.3
+const ABSOLUTE_URL_REGEX = /^[a-zA-Z][a-zA-Z\d+\-.]*?:/;
```

Comment:
The regexes are simple and I could not see any vulnerabilities reports or significant changes in the last two years.